wcfsetup/install/files/lib/data/user/User.class.php

```diff
@@ -732,7 +732,7 @@ public function mustSelfEmailConfirm()
 
     /**
      * Returns true if the user is a member of a user group that requires
-     * multi-factor authentication to be enabled.
+     * multifactor authentication to be enabled.
```

Comment:
This change is incorrect: https://en.wikipedia.org/wiki/Multi-factor_authentication

Reply:
Different websites, different opinions ;

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/multifactor-authentication